routers/api/v1/repo/file.go

```diff
@@ -67,6 +67,8 @@ func GetRawFile(ctx *context.APIContext) {
 	// responses:
 	//   200:
 	//     description: Returns raw file content.
+	//     schema:
+	//       type: file
```

Comment:
Is this a valid value for `type`?

https://json-schema.org/understanding-json-schema/reference/type

Reply:
It's a raw file content, not json schema. See,

https://swagger.io/docs/specification/2-0/describing-responses/#response-that-returns-a-file